### ISSUE TITLE
aria-label overwriting accessible names for accordion buttons

### ISSUE THREAD

**Issue by scottaohara** [stuartjnelson/badger-accordion#17]
# Expected Behavior
The visible label / unique accessible name of each button should be what's announced to screen readers when buttons are focused.

# Current Behavior
"Open accordion panel" is the accessible name of all accordion panel triggers.

## Steps to Reproduce
1. Turn on a screen reader
2. Navigate to an accordion trigger
3. The `aria-label` overwrites the accessible name of the button.

## Fix
The `aria-label`s should be removed from these buttons. 
1. Because they're overwriting the accessible name of the button
2. They're unnecessary as `aria-expanded` already handles the appropriate announcement of state.

**Comment by stuartjnelson** [comment]
Thanks, @scottaohara! Sorry it took me so long to implement you're 2 tickets.
